Documentation/ApiOverview/UpdateWizards/Creation.rst

```diff
@@ -159,13 +159,58 @@ Registering wizards
 Once the wizard is created, it needs to be registered. Registration is done in
 :file:`ext_localconf.php`::
 
-   $GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['ext/install']['update']['exampleUpdateWizard']
+   $GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['ext/install']['update']['myExtension_exampleUpdateWizard']
       = \Vendor\ExtName\Updates\ExampleUpdateWizard::class;
 
-**Important:** Use the same identifier as key (here: `exampleUpdateWizard`), which
+**Important:** Use the same identifier as key (here: `myExtension_exampleUpdateWizard`), which
 is returned by :php:`UpgradeWizardInterface::getIdentifier()` in your wizard
 class.
 
+.. index:: Upgrade wizards; Identifier
+.. _upgrade-wizards-identifier:
+
+Wizard identifier
+=================
+
+The wizard identifier is used:
+
+*  when calling the wizard from the :ref:`command line <upgrade_wizard_execute>`.
+*  when marking the wizard as done in the table `sys_registry`
+
+Since all upgrade wizards of TYPO3 core and extensions are registered using the
+identifier as key in the global array
+:php:`$GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['ext/install']['update']`, it
+is recommended to prepend the identifier with the extension key to make it unique.
```

Comment:
Here we should find a better term than "extension key" since that's not really true. The "extension key" would be e.g. `foo_bar_qux`. But we suggest `fooBarQux` instead here.